cheroot/ssl/builtin.py

```diff
@@ -272,63 +272,20 @@ def bind(self, sock):
 
     def wrap(self, sock):
         """Wrap and return the given socket, plus WSGI environ entries."""
-        EMPTY_RESULT = None, {}
         try:
             s = self.context.wrap_socket(
                 sock, do_handshake_on_connect=True, server_side=True,
             )
+            return s, self.get_environ(s)
```

Comment:
There must be exactly one statement in a try-block. Otherwise, await surprises. For example, `self.get_environ()` call would get processed by the except blocks here, while one would only expect the `wrap_socket()`'s errors to be handled in this place.

Reply:
I moved it into a try/else